### PR TITLE
Styling updates to tabs, explore panel, action buttons, and code font-size

### PR DESCRIPTION
**Pull request by taiyab**
## Context

This started out as just fixing tab colors, but later evolved into making a bunch of styling updates throughout.

Requires careful attention to the global changes: monospace/code font-sizes (this needs to be thoroughly reviewed wherever we display code), but also all the smaller, local changes.

| Before    | After |
| -------- | ------- |
| CleanShot 2024-07-25 at 13 48 42@2x    | CleanShot 2024-07-25 at 13 47 16@2x       |
| CleanShot 2024-07-25 at 13 49 38@2x | CleanShot 2024-07-25 at 13 51 09@2x     |

## Test plan
Manually tested locally for visual changes.

### CODE REVIEW

File: client/web-sveltekit/src/lib/TabsHeader.svelte [hunk 1]
```diff
@@ -62,7 +62,7 @@
 
 <style lang="scss">
     .tabs-header {
-        --icon-color: var(--header-icon-color);
+        --icon-color: var(--icon-color);
```
Comment (camdencheek) [inline client/web-sveltekit/src/lib/TabsHeader.svelte:65]: This changes it to be effectively a no-op, so we should just remove the line if that's intentional

File: client/web-sveltekit/src/lib/codenav/ExplorePanel.svelte [hunk 2]
```diff
@@ -412,7 +412,6 @@
         justify-content: center;
         height: 100%;
         width: 100%;
-        color: var(--text-muted);
```
Comment (camdencheek) [inline client/web-sveltekit/src/lib/codenav/ExplorePanel.svelte:415]: These are departures from the designs, right? What's the process for reconciling these?

File: client/web-sveltekit/src/lib/repo/Permalink.svelte [hunk 3]
```diff
@@ -43,6 +43,8 @@
 
 <style lang="scss">
     a {
+        --icon-color: var(--icon-color);
```
Comment (camdencheek) [inline client/web-sveltekit/src/lib/repo/Permalink.svelte:46]: This is a no-op

File: client/web-sveltekit/src/routes/[...repo=reporev]/(validrev)/(code)/-/blob/[...path]/OpenInCodeHostAction.svelte [hunk 4]
```diff
@@ -38,6 +38,8 @@
 
 <style lang="scss">
     a {
+        --icon-color: var(--icon-color);
```
Comment (camdencheek) [inline client/web-sveltekit/src/routes/[...repo=reporev]/(validrev)/(code)/-/blob/[...path]/OpenInCodeHostAction.svelte:41]: Another no-op

File: client/web-sveltekit/src/routes/svelte-overrides.scss [hunk 5]
```diff
@@ -15,10 +15,10 @@ body {
     --monospace-font-family: 'Roboto Mono Variable', monospace;
     --code-font-family: var(--monospace-font-family);
     --code-line-height: 1.54;
+    --font-size-lg: 1rem;
     --font-size-base: 0.9375rem;
     --font-size-small: 0.875rem;
-    --font-size-extra-small: 0.6875rem;
-    --font-size-tiny: 0.8125rem;
```
Comment (camdencheek) [inline client/web-sveltekit/src/routes/svelte-overrides.scss:21]: Looks like there are a couple of spots where `--font-size-tiny` is still used

**Review by camdencheek — CHANGES_REQUESTED** [review]

File: client/web-sveltekit/src/lib/codenav/ExplorePanel.svelte (hunk 2 above)
Comment (taiyab) [reply client/web-sveltekit/src/lib/codenav/ExplorePanel.svelte:415]: Decided to revert the change here instead and make sure the icon color is correct.

File: client/web-sveltekit/src/routes/svelte-overrides.scss (hunk 5 above)
Comment (taiyab) [reply client/web-sveltekit/src/routes/svelte-overrides.scss:21]: Should be cleared now.

**Pull request closed without merging**